core/src/test/scala/unit/kafka/log/LogValidatorTest.scala

```diff
@@ -1200,6 +1193,16 @@ class LogValidatorTest {
     MemoryRecords.readableRecords(buf.slice())
   }
 
+  private def createDiscontinuousOffsetRecords(magicValue: Byte,
```

Comment:
I missed this before, but this is not used.

Reply:
Ack.